Library/Formula/libdecnumber.rb

```diff
@@ -0,0 +1,41 @@
+class Libdecnumber < Formula
+  desc "ANSI C implementation of General Decimal Arithmetic"
+  homepage "http://speleotrove.com/decimal/#decNumber"
+  url "http://download.icu-project.org/files/decNumber/decNumber-icu-368.zip"
+  version "3.68"
+  sha256 "14ec2cf30b58758493a7661b78b80abfb281652b61a425b85cda83173518fe25"
+  resource "cmakelists" do
```

Comment:
There is *no* Makefile in the distribution. How do you suggest I should build the libraries?